components/script/page.rs

```diff
@@ -95,6 +95,8 @@ pub struct Page {
     /// An enlarged rectangle around the page contents visible in the viewport, used
     /// to prevent creating display list items for content that is far away from the viewport.
     pub page_clip_rect: Cell<Rect<Au>>,
+
+    pub send_notifications: Cell<bool>,
```

Comment:
Let's call this `devtools_want_updates` instead, and document it with something like "Whether the developer tools have requested live updates of page changes."